website/docs/d/container_registry.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_container_registry"
+sidebar_current: "docs-azurerm-datasource-image"
+description: |-
+  Get information about an Image
+
+---
+
+# Data Source: azurerm_container_registry
+
+Use this data source to access information about a Container Registry.
+
+## Example Usage
+
+```hcl
+data "azurerm_container_registry" "test" {
+  name                = "testacr"
+  resource_group_name = "test"
+}
+
+output "image_id" {
+  value = "${data.azurerm_container_registry.test.id}"
+}
```

Comment:
_minor_ could we make this `login_server ` which would be a more likely output here? e.g.

```
output "login_server" {
  value = "${data.azurerm_container_registry.test.login_server}"
}
```